packages/cow/cow.2.3.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+maintainer: "Anil Madhavapeddy <anil@recoil.org>"
+authors: ["Anil Madhavapeddy" "Thomas Gazagnaire" "David Sheets"
+          "Rudi Grinberg" "Timothy Bourke"]
+homepage: "https://github.com/mirage/ocaml-cow"
+doc: "http://mirage.github.io/ocaml-cow/"
+license: "ISC"
+dev-repo: "https://github.com/mirage/ocaml-cow.git"
+bug-reports: "https://github.com/mirage/ocaml-cow/issues"
+tags: ["org:mirage" "org:xapi-project" "www" "html" "xml"
+       "css" "json" "markdown" ]
+
+available: [ ocaml-version >= "4.02.3"]
```

Comment:
There is a failure in the CI due to the use of String.lowercase_ascii that was introduced in 4.03